webrtc-stats.html

```diff
@@ -1143,9 +1143,20 @@
              boolean                       readable;
              unsigned long long            bytesSent;
              unsigned long long            bytesReceived;
-             double                        roundTripTime;
+             double                        averageRtt;
+             double                        currentRtt;
```

Comment:
At the interim where we discussed this, I thought we decided on having a "totalRtt" that was the sum of all round trip times? That would allow you to compute the overall average over the whole session by dividing by responsesReceived, but also allows you to compute the average over a specific time interval. Did that decision change?